requirements/requirements.txt

```diff
@@ -72,7 +72,6 @@ twilio==3.6.4
 django-countries==3.3
 reportlab==3.0
 stripe==1.12.2
-mailchimp==2.0.7
```

Comment:
should we add this to `uninstall-requirements.txt`? i think otherwise it will still sit in our prod venvs